arwen/elrondapi/manBufOps.go

```diff
@@ -119,11 +119,6 @@ func ManagedBufferImports(imports *wasmer.Imports) (*wasmer.Imports, error) {
 		return nil, err
 	}
 
-	imports, err = imports.Append("mBufferSetBytes", v1_4_mBufferSetBytes, C.v1_4_mBufferSetBytes)
```

Comment:
Why was `mBufferSetBytes()` removed?

Reply:
Was duplicate! 
Discovered because I actually added a check for duplicates!